contrib/masterfiles-stage/README.org

```diff
@@ -59,20 +59,30 @@ earlier versions of git may or may not work.
 :PROPERTIES:
 :ID:       2aeaaa9b-1229-4c14-b130-6d86e370de42
 :END:
-Installation simply consists of saving =masterfiles-stage.sh= and its
-supporting =common.sh= to the same directory on your policy server
-and making sure that =masterfiles-stage.sh= is executable.
-
-#+BEGIN_EXAMPLE
-  wget https://raw.githubusercontent.com/cfengine/core/master/contrib/masterfiles-stage/masterfiles-stage.sh -O /var/cfengine/bin/masterfiles-stage.sh
-  wget https://raw.githubusercontent.com/cfengine/core/master/contrib/masterfiles-stage/common.sh -O /var/cfengine/bin/common.sh
-  chown root:root /var/cfengine/bin/masterfiles-stage.sh /var/cfengine/bin/common.sh
-  chmod 555 /var/cfengine/bin/masterfiles-stage.sh
-  chmod 444 /var/cfengine/bin/common.sh
-#+END_EXAMPLE
+Installation simply consists of saving =masterfiles-stage.sh= and its supporting
+=common.sh= to the same directory on your policy server, making sure that
+=masterfiles-stage.sh= is executable and configuring the default parameter set.
+
+*Note:* These instructions install the tooling into the same location where they
+are pre-installed when using CFEngine Enterprise.
+
+#+CAPTION: Installing masterfiles-stage
+#+BEGIN_SRC sh
+  SRC_DIR="https://raw.githubusercontent.com/cfengine/core/master/contrib/masterfiles-stage/"
+  DC_SCRIPTS_DIR=$(cf-promises --file update.cf --show-vars=dc_scripts | awk '/update_def\.dc_scripts/ {print $2}')
+  DEFAULT_PARAMS_FILE="/opt/cfengine/dc-scripts/params.sh"  
+  mkdir -p ${DC_SCRIPTS_DIR}
+  mkdir -p ${DEFAULT_PARAMS_FILE}
```

Comment:
it will create `params.sh` directory. You need `dirname` here.

Reply:
ah yes, thanks!

Reply:
Fixed.